app/src/main/res/values-de-rDE/strings.xml

```diff
@@ -0,0 +1,185 @@
+<resources>
+    <string name="app_name" translatable="false">PeakAR</string>
+
+    <!-- Google -->
+    <!-- Sign-in status messages -->
+    <string name="mountain_info" translatable="false">Mont Blanc - Monte Bianco lat: 45.8325 lon: 6.8641666666667 alt: 4810 <br /><br />
+                                 Dent du Geant lat:45.86355980599387 lon:6.951348205683087 alt: 4013<br /><br />
+                                 Aiguille du plan lat:45.891667 lon:6.907222 alt: 3673<br /><br />
+                                 Pointe de Lapaz lat:45.920774986207014 lon:6.812914656881065 alt: 3660</string>
+    <!-- Sign-in button labels and other strings -->
+
+    <string name="title_activity_main_menu" translatable="false">MainMenuActivity</string>
+
+    <!-- BUTTON TITLES -->
+    <string name="start_button">Start Peak AR</string>
+    <string name="collection_button">Collection</string>
+    <string name="ranking_button">Ranking</string>
+    <string name="gallery_button">Gallery</string>
+    <string name="map_button">Map</string>
+    <string name="sign_in_button">Sign In</string>
+    <string name="sign_out_button">Sign Out</string>
+    <string name="sign_up_button">Sign Up</string>
+
+    <!-- USERNAME CHOICE -->
+    <string name='already_existing_username'>The username is already used by another user. Choose a new one.</string>
+    <string name='invalid_username'>The username is not valid. It must contain between 3 and 15 alphanumeric characters.</string>
+    <string name='current_username'>You can\'t choose the username you already have!</string>
+    <string name='available_username'>Your username has changed!</string>
+    <string name='insert_username_button'>Insert your username here</string>
+    <string name='change_username_button'>Change username</string>
+    <string name='submit_username_button'>Change</string>
+    <string name='registered_username' translatable="false">Welcome to PeakAR community!</string>
+
+    <!-- FRIENDS SYSTEM -->
+    <string name='add_friend_button'>Add a friend</string>
+    <string name='friends_button'>Friends</string>
+    <string name='insert_friend_button' translatable="false">Insert here</string>
```

Comment:
Cirrus complains that this is translated in German. Since it is set to false it shouldn't